far/editor.cpp

```diff
@@ -243,6 +243,15 @@ void Editor::ShowEditor()
 	Color = colors::PaletteColorToFarColor(COL_EDITORTEXT);
 	SelColor = colors::PaletteColorToFarColor(COL_EDITORSELECTEDTEXT);
 
+	// Calculate line number column width if needed
+	int LineNumWidth = 0;
```

Comment:
Can we please reword it as 

```C++
const auto LineNumWidth = EdOpt.ShowLineNumbers?
    std::max(3, static_cast<int>(std::log10(Lines.size())) + 1) + 1 :
    0;
```
- const
- +1 for spacing is more explicit and can be changed in one place if needed.